validator/src/main/resources/expected-data-template/springboot/springbootAppExpectedAWSSDKTrace.mustache

```diff
@@ -0,0 +1,58 @@
+[{
+  "name": "/aws-sdk-call",
+  "fault": false,
+  "error": false,
+  "throttle": false,
+  "http": {
+    "request": {
+      "url": "{{endpoint}}/aws-sdk-call",
+      "method": "GET"
+    },
+    "response": {
+      "status": 200
+    }
+  },
+  "subsegments": [
+    {
+      "fault": false,
+      "error": false,
+      "throttle": false,
+      "subsegments": [
+        {
+          "name": "S3",
+          "fault": false,
+          "error": false,
+          "throttle": false,
```

Comment:
can we remove the throttle from here?

Reply:
Should've removed them from the start, they've been removed!